src/deepsparse/pipelines/custom_pipeline.py

```diff
@@ -148,6 +178,27 @@ def process_engine_outputs(
         """
         return self._process_outputs_fn(engine_outputs, **kwargs)
 
+    def _read_processing_functions_from_file(self, processing_file: str):
+        """
+        Parses the file containing the `preprocess` and `postprocess` functions
+
+        :pre-condition: The file is a valid `.py` file that exists and may
+            contain a preprocess and a postprocess function
+        :param processing_file: The path to the file containing the preprocess
+            and postprocess functions
+        :return: The preprocess and postprocess functions from the file
+        """
+        print(
```

Comment:
We should use logger over print

Reply:
Seemed unnecessary to add a logger for just one log, but good call, added!